src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1692,7 +1694,12 @@ int xcldev::device::testP2p()
     }
 
     for (int32_t i = 0; i < map->m_count && ret == 0; i++) {
-        const std::vector<std::string> supList = { "HBM", "DDR", "bank" };
+        std::vector<std::string> supList = { "HBM", "bank" };
+
+        //p2p is not supported for DDR on u280
+        if(vbnv.find("u280") == std::string::npos)
```

Comment:
you probably want to search for "_u280_" here?

Reply:
I'm doing if not u280 then add DDR to the "supList"